Index.html

```diff
@@ -62,7 +62,7 @@ <h1>Welcome to Erika Taege's CSD120 Website!</h1>
         </table>
         <h2>My Favourite Things</h2>
         <a href="https://tsm.gg/teams/lol" target="_blank">
-          <figure></figure>
+          <figure>
```

Comment:
A few problems from previous labs.  Make sure you're pulling in my suggested changes so that you have a clean slate to work with for the next lab